### PR TITLE
Add recipe image feature

### LINKED ISSUE #99: Add recipe image feature

### PR DESCRIPTION
**Pull request by jiaqi20**

### CODE REVIEW

**Review by caitlinjee — APPROVED** [review]
lgtm

**Review by daongochieu2810 — APPROVED** [review]

File: src/main/java/seedu/address/logic/parser/AddRecipeCommandParser.java [hunk 1]
```diff
@@ -48,10 +49,40 @@ public AddRecipeCommand parse(String args) throws ParseException {
...
+            out.close();
+            in.close();
+            byte[] response = out.toByteArray();
 
+            recipeImage = this.getClass().getResource("/images").getPath() + filename;
+            FileOutputStream fos = new FileOutputStream(recipeImage);
+            fos.write(response);
+            fos.close();
+        }
+         */
```
Comment (TianYong-Goh) [inline src/main/java/seedu/address/logic/parser/AddRecipeCommandParser.java:84]: Just curious is this the Url part?

**Review by TianYong-Goh — APPROVED** [review]
lgtm

**Review by oliviajohansen — APPROVED** [review]
lgtm

File: src/main/java/seedu/address/logic/parser/AddRecipeCommandParser.java (hunk 1 above)
Comment (jiaqi20) [reply src/main/java/seedu/address/logic/parser/AddRecipeCommandParser.java:84]: Yes it is haha to save into local but I realized it works without this code so I commented it out first

**Pull request merged**